### ISSUE TITLE
How to publish on Yandex?

### ISSUE THREAD

**Issue by subsoap** [indiesoftby/defold-yagames#3]
Google won't tell me how :D

**Comment by alexeyfeskov** [comment]
Hmm.. It looks like the landing page is only in Russian https://yandex.ru/dev/games/
But there are EN docs https://yandex.ru/dev/games/doc/dg/concepts/about.html/?lang=en

**Comment by aglitchman** [comment]
Yes, there are docs in English. The dashboard is also available in English - https://games.yandex.ru/console

Yandex welcomes developers from any countries!

**Comment by aglitchman** [comment]
Defold Blog: _Releasing HTML5 games on Yandex.Games_ - https://defold.com/2021/04/21/Releasing-games-on-Yandex/
